.github/CONTRIBUTING.md

```diff
@@ -1 +1,6 @@
-Please refer to the [developers guide](https://matplotlib.org/devel/index.html).
+Please refer to the [contributing guide](https://matplotlib.org/devel/index.html).
+
+
+
+
+
```

Comment:
Please could you remove all the added blank lines?

Reply:
Okay, I'll do that.